src/suricata.h

```diff
@@ -178,6 +178,8 @@ extern volatile uint8_t suricata_ctl_flags;
 extern int g_disable_randomness;
 extern uint16_t g_vlan_mask;
 
+int InitGlobal(void);
```

Comment:
can you move this to line 196 below. Seems out of place here.

Reply:
ok